File: Swift/String/551_attendance_record_one.swift
class Solution {
    func checkRecord(_ s: String) -> Bool {        
        // Count number of 'A's in s
        let absent = s.filter { $0 == "A" }.count
        // If more than two continuous 'L'
        let late = s.contains("LLL")
        
        return (absent <= 1) && !late
    }
}

File: Python/String/551_attendance_record_one.py
class Solution:
    def checkRecord(self, s: str) -> bool:
        if (len(s) > 1):
            if ('A' in s or 'L' in s):
                # More than two continuous 'L'
                if ('LLL' in s):
                    return False
                if ('A' in s):
                    if (s.find('A') < len(s) - 1 and 'A' in s[s.find('A') + 1:]):
                        return False
        return True
